### PR TITLE
Landing Page Layout

### PR DESCRIPTION
**Pull request by madhurimt**
Tried a basic landing page layout.
image

-Concepts Learnt 
-Creating NavBar using FlexBox
-Positioning 

tried using class="fa-solid fa-arrow-up-right"> in button but it is not working.

### CODE REVIEW

**Comment by madhurimt** [comment]
Issues:-
- Tried using "<i class="fa-solid fa-arrow-up-right"></i> " but it is not working.

**Comment by jsjayfiled** [comment]
> 

Those are classes that will display icons on the page.  They are from the 'font awesome' icon library.  Using the library used to be as easy as adding a <link> tag in your <head> then putting classes like "fa-solid fa-arrow-up-right" would give you an icon, but now they want you to sign up and use a different method so I'd avoid them.

But if you want icons I can suggest 'line icons' - [check them out](https://lineicons.com/docs/)

File: style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*{
+    padding:0;
+    margin: 0;
+    box-sizing: border-box;
+}
```
Comment (jsjayfiled) [inline style.css:7]:
```suggestion
} add a space between the selector (in this case the asterisk *) and the opening brackets - on all css rules
```

**Review by jsjayfiled — CHANGES_REQUESTED** [review]

**Comment by jsjayfiled** [comment]
Looks good!
What else are you planning to do to it!

Also, just letting you know that if I am taking too long to get back to you ( I can't check every day 🥹 ) you can merge these yourself.

You are doing really well and should be able to start JS soon :-)

File: style.css [hunk 2]
```diff
@@ -44,13 +45,35 @@ li a{
...
+
+img{
+    position:absolute;
+    right:0;
+    top:10vh;
+    height:650px;
+    width:900px;
+    object-fit: cover;
+    padding:10px;
+    border:1px solid #c9a356;
```
Comment (jsjayfiled) [inline style.css]: The design showed a dashed border, you can change this to dashed to get the desired result.

Or if you want to get real fancy you can remove the border property and add in this property instead - it's a little trick to have more control over the dashes

background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23333' stroke-width='4' stroke-dasharray='6 14' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");

stroke-width is a good one to change and the dasharray is how many lines followed by how many gaps

**Review by jsjayfiled — APPROVED** [review]

**Pull request merged**